src/platform-includes/crons/setup/java.spring-boot.mdx

```diff
@@ -1,4 +1,4 @@
-If you are using [Quartz](http://www.quartz-scheduler.org/), please see <PlatformLink to="/configuration/integrations/quartz/">our Quartz integration</PlatformLink>. You may also [send check-ins manually](/platforms/java/crons/).
+If you are using [Quartz](http://www.quartz-scheduler.org/), please see <PlatformLink to="/integrations/quartz/">our Quartz integration</PlatformLink>. You may also [send check-ins manually](/platforms/java/crons/).
```

Comment:
Link 404s because "Integrations" is missing for Spring Boot, maybe others as well.